docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -32,7 +32,7 @@ Python shebang & UTF-8 coding
 
 Begin your Ansible module with ``#!/usr/bin/python`` - this "shebang" allows ``ansible_python_interpreter`` to work. Follow the shebang immediately with ``# -*- coding: utf-8 -*-`` to clarify that the file is UTF-8 encoded.
 
-.. note:: Using ``#!/usr/bin/env``, makes ``env`` the interpreter and bypasses ``ansible_<interpreter>_interpreter`` logic.
+.. warning:: Using ``#!/usr/bin/env``, makes ``env`` the interpreter and bypasses ``ansible_<interpreter>_interpreter`` logic. Using arguments in combination with ``env`` (e.g. ``#!/usr/bin/env python``) does not work.
```

Comment:
It is not just env, it is with any interpreter, i would make this a separate warning. 
```
.. warning:: 
  - Using ``#!/usr/bin/env``, makes ``env`` the interpreter and bypasses ``ansible_<interpreter>_interpreter`` logic.    
  -  Passing arguments to the interpreter in the shebang also does not work.
```

Reply:
Thanks. I've updated it.